src/server/svr_jobfunc.c

```diff
@@ -437,7 +437,7 @@ svr_enquejob(job *pjob)
 	/* update any entity count and entity resources usage for the queue */
 
 	if (!(pjob->ji_qs.ji_svrflags & JOB_SVFLG_SubJob) || (server.sv_attr[(int)SRV_ATR_State].at_val.at_long == SV_STATE_INIT)) {
-		account_entity_limit_usages(pjob, pque, NULL, INCR);
+		account_entity_limit_usages(pjob, pque, NULL, INCR, ETLIM_ACC_ALL);
```

Comment:
Here we have to `INCR` all the 4 limit attributes. Hence passing `ETLIM_ACC_ALL`